bundles/org.openhab.binding.meteoalerte/src/main/java/org/openhab/binding/meteoalerte/internal/handler/MeteoAlerteHandler.java

```diff
@@ -19,19 +19,20 @@
 import java.net.MalformedURLException;
 import java.nio.charset.StandardCharsets;
 import java.time.ZonedDateTime;
-import java.util.Map;
 import java.util.Objects;
+import java.util.Optional;
 import java.util.concurrent.ScheduledFuture;
 import java.util.concurrent.TimeUnit;
 
+import javax.ws.rs.HttpMethod;
```

Comment:
I am not sure that this is the more appropriate import.
I know that a similar enum is also available in Jetty.

Reply:
Let's roll this back. HttpUtils does not accept HttpMethod.GET directly.